pkg/api/org_invite.go

```diff
@@ -119,24 +132,24 @@ func (hs *HTTPServer) AddOrgInvite(c *contextmodel.ReqContext) response.Response
 			Template: "new_user_invite",
 			Data: map[string]interface{}{
 				"Name":      util.StringsFallback2(cmd.Name, cmd.Email),
-				"OrgName":   c.OrgName,
-				"Email":     c.Email,
+				"OrgName":   c.SignedInUser.GetOrgID(),
```

Comment:
question: is org name the same as org ID?

Reply:
Or should it be
```suggestion
				"OrgName":   c.SignedInUser.GetOrgName(),
```

Reply:
:innocent: autocompletion you betray me